images/vli/sle12_sp4/config.kiwi

```diff
@@ -32,7 +32,7 @@
         </type>
     </preferences>
     <preferences profiles="Devel">
-        <type image="oem" filesystem="xfs" bootloader="grub2" bootloader_console="console" initrd_system="dracut" kernelcmdline="splash=verbose loglevel=3 nomodeset net.ifnames=1 numa_balancing=disable transparent_hugepage=never intel_idle.max_cstate=1 processor.max_cstate=1 crashkernel=232M,high crashkernel=80M,low nohz=off skew_tick=1 intel_pstate=disable clocksource=tsc console=ttyS0,115200 rd.kiwi.debug" firmware="uefi" bootpartition="false">
+        <type image="oem" filesystem="xfs" bootloader="grub2" bootloader_console="console" initrd_system="dracut" kernelcmdline="splash=verbose loglevel=3 nomodeset net.ifnames=1 numa_balancing=disable transparent_hugepage=never intel_idle.max_cstate=1 processor.max_cstate=1 crashkernel=232M,high crashkernel=80M,low nohz=off skew_tick=1 intel_pstate=disable clocksource=tsc console=ttyS0,115200 disable_mtrr_trim tsc=reliable DefaultTimeoutStartSec=900s DefaultTimeoutStopSec=900s Nmi_watchdog=0 rd.kiwi.debug" firmware="uefi" bootpartition="false">
```

Comment:
@schaefi sorry for back & forth on this one, in our latest decision, we have deferred provision of this image on SLES12 SP4 on the new hardware.
We will need it for just the SLES15 train as of now.
Can you please change, sorry for the inconvenience.
Thanks !!

Reply:
:) no problem. That's why we have pull requests and a conversation. I'll adapt

Reply:
done, changes applied to SLE15 VLI images. Everything else stays untouched. Let me know if that is correct. Thanks